testproject/tests/test_second_step_authentication.py

```diff
@@ -431,17 +388,17 @@ def test_change_primary_method(active_user_with_many_otp_methods):
     response = client.post(
         path="/auth/mfa/change-primary-method/",
         data={
-            "method": "sms",
-            "code": create_otp_code(first_primary_method.secret),
+            "method": "sms_twilio",
+            "code": handler.create_code(),
         },
         format="json",
     )
     new_primary_method = active_user.mfa_methods.filter(
         is_primary=True,
     )[0]
-    assert response.status_code == 200
-    assert first_primary_method != new_primary_method
-    assert new_primary_method.name == "sms"
+    assert response.status_code == 204
```

Comment:
in other tests we are using `rest_framework.status` const so we can do in this test too